### PR TITLE
Implementation of px->pt conversion

### PR DESCRIPTION
**Pull request by ChriPiv**

### CODE REVIEW

File: src/android/com/mirasense/scanditsdk/plugin/UIParamParser.java [hunk 1]
```diff
@@ -221,10 +225,37 @@ public static boolean bundleContainsBundleKey(Bundle bundle, String key) {
...
-                Log.e("ScanditSDK", "array contains wrong class - " + obj.getClass().getName());
                 return false;
             }
         }
         return true;
     }
+
+    // Converts px to dp if String ends with 'px'
+    public static Integer getDp(Object obj) {
+        if (obj instanceof Number) {
```
Comment (biasmv) [inline src/android/com/mirasense/scanditsdk/plugin/UIParamParser.java]: This cast isn't safe. obj could also be a double/float.

File: src/android/com/mirasense/scanditsdk/plugin/UIParamParser.java [hunk 2]
```diff
@@ -221,10 +225,37 @@ public static boolean bundleContainsBundleKey(Bundle bundle, String key) {
...
+            return (Integer) obj;
```
Comment (biasmv) [inline src/android/com/mirasense/scanditsdk/plugin/UIParamParser.java]: This cast isn't safe. obj could also be a double/float.

**Comment by biasmv** [comment]
Thanks, apart from the small comment above, this looks good.

**Comment by ChriPiv** [comment]
This should cover all cases

**Comment by ChriPiv** [comment]
Is rendered unecessasry because of new pull request #45

**Pull request closed without merging**